portfolio/src/main/java/com/google/sps/servlets/DeleteDataServlet.java

```diff
@@ -17,10 +19,26 @@
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("text/plain;");
+    UserService userService = UserServiceFactory.getUserService();
     long id = Long.parseLong(request.getParameter("id"));
     
     Key commentKey = KeyFactory.createKey("Comment", id);
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-    datastore.delete(commentKey);
+
+    String posterEmail = "";
+    String userEmail = userService.getCurrentUser().getEmail();
+    try {
+      posterEmail = (String) datastore.get(commentKey).getProperty("email");
+    } catch (Exception e) {
+      System.err.println(e);
```

Comment:
I am curious to know where this log is written to? Is there to have a permanent log for your web application. This looks like it is writing to a console but where is that concole?

Reply:
Maybe we should return an error to the user?

Reply:
Yes, I need to return the error to user.